src/renderer/components/ObservationCard.tsx

```diff
@@ -42,9 +42,7 @@ function ObservationCard(props: ObservationCardProps) {
   };
 
   const handlePredictionOverride = (newPrediction: CreatableOption | null) => {
-    if (newPrediction === null || newPrediction.value !== topPrediction.value) {
```

Comment:
Now if we select the top prediction from the override dropdown it will be considered as overridden; this condition prevented it. I think it is acceptable to handle it this way now though; I want to refactor how prediction overrides are stored and it will change a couple of things.